### PR TITLE
Typo fixes + code readability improvements

### PR DESCRIPTION
**Pull request by jaceklaskowski**

### CODE REVIEW

File: core/src/main/scala/org/apache/spark/rdd/HadoopRDD.scala [hunk 1]
```diff
@@ -195,9 +196,6 @@ class HadoopRDD[K, V](
     // add the credentials here as this can be called before SparkContext initialized
     SparkHadoopUtil.get.addCredentials(jobConf)
     val inputFormat = getInputFormat(jobConf)
-    if (inputFormat.isInstanceOf[Configurable]) {
```
Comment (srowen) [inline core/src/main/scala/org/apache/spark/rdd/HadoopRDD.scala]: Why can this be removed?

**Comment by srowen** [comment]
Except one comment, all the changes look fine, but I'm not sure I think it's a great use of time. Most of these are very marginal clarifications or spacing fixes. I'd prefer to focus on more substantive changes, as mentioned several times before. How can we do that?

File: core/src/main/scala/org/apache/spark/rdd/HadoopRDD.scala (hunk 1 above)
Comment (jaceklaskowski) [inline core/src/main/scala/org/apache/spark/rdd/HadoopRDD.scala]: It's a duplication since `getInputFormat` does exactly this.

**Comment by jaceklaskowski** [comment]
Thanks @srowen for reviewing the changes. They indeed are small since they're the outcome of me going through the code and learning Spark this way. When I run across a typo or similar inconsistency, I fix it and move on.

If you've got a task at hand I could work on - something small(ish) - I'd love to hear about it. Guide me.

**Comment by srowen** [comment]
Typos, maybe, or doc fixes. Some of these aren't fixes, or are just whitespace changes. It's not worth the time, really, unless you're cleaning up along the way to another change or are zapping a bunch of instances in one go.

Look at the `starter` label. This is answered in https://cwiki.apache.org/confluence/display/SPARK/Contributing+to+Spark

**Comment by jaceklaskowski** [comment]
The only files that touch spaces are `ShuffleMapTask.scala` and `TaskSet.scala` (and these could be reverted). The others look more than space fixes.

What should I do with the changes?

On to the starter tag...

**Comment by srowen** [comment]
Merged to master

**Comment by jaceklaskowski** [comment]
Thanks again @srowen! Next pull req should be from starter tag.

**Pull request closed without merging**